react/Hero/styles.styl

```diff
@@ -0,0 +1,37 @@
+@require 'settings/breakpoints.styl'
+
+.Hero
```

Comment:
I don't mean to be fussy but wouldn't it be better to have the same syntax of CSS all over Cozy-UI?
I mean like lowercase only for classes and `hero-title` instead of `Hera__title`

Reply:
OK, I verified in the styleguide and it's true that "We use BEM naming convention but with a slight difference, we use this syntax block-element--modifier." :-)

I'll change this but for the record : I am not particularly a fan of following a standard and changing it just a bit. It is not written why there is this change in the CSS styleguide.

One of the advantage (other that doing like everyone else) I see in keeping with BEM for the Block__Element part instead of having Block-Element is that we can write `style={styles.Block__Element}`. With the homemade syntax we have to write `style={styles['Block-Element']}` since dashes would be interpreted as minus signs. It does not work for modifier though :/

I look forward to the day where we use BEM instead cozy-BEM.

Reply:
For what it's worth BEM is not about the syntax, it's about the methodology. 
The syntax "by default" provided by BEM has never been mandatory, it's purely arbitrary, and they say it openly. It's more a "Choose a meaningful syntax and stick to it" than a "Choose my syntax and stick to it".
Respecting the BEM's example like a dogma makes no sense to me. 
Not that I absolutely want to keep the way it is so far in Cozy-UI, but I'll need a little better reason than "everybody does like the example" to change everything we did so far. 😕

Reply:
done

Reply:
I changed it.

> Respecting the BEM's example like a dogma makes no sense to me.

This is why I gave a reason that makes sense to me since we use CSS modules.

To me, the case to defend is why we've did differently than the others. We are a framework : if we do something else than everybody else, people will have a harder time to learn and adopt it. The change has to bring value to be meaningful.

I also agree that changing it now is not a good idea since a lot of styles are still used manually (instead of via components). This is why I changed it in my PR.

Reply:
Well I don't really agree with everything you said but we could just discuss it on a proper medium later with pleasure. 😄